torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -536,6 +565,46 @@ struct GraphFuser {
     return true;
   }
 
+  void insertExplicitExpands(Node * node) {
+    JIT_ASSERT(isSimpleMap(node));
+    auto & map_size = node->output()->type()->cast<TensorType>()->sizes();
+    auto * graph = node->owningGraph();
+
+    WithInsertPoint guard(node);
+
+    auto it_end = node->inputs().end();
+    if (node->matches("aten::add(Tensor self, Tensor other, *, Scalar alpha) -> Tensor", /*const=*/attr::alpha)) {
```

Comment:
This seems fragile. Someone might add another node that has a scalar, and then forget to update this here.